src/main/groovy/com/ullink/ProjectFileParser.groovy

```diff
@@ -44,21 +44,42 @@ class ProjectFileParser {
         FilenameUtils.removeExtension(file) + newExtension
     }
 
+    File getOutputPath() {
+        if (msbuild.destinationDir) {
+            project.file(msbuild.destinationDir)
+        }
+        if (properties.TargetDir) {
+            new File (properties.TargetDir)
+        }
+    }
+
     File getDotnetAssemblyFile() {
-        project.file(properties.TargetPath)

```

Comment:
Hmmm not sure I get that, the solution parser should only be an intermediate to gather all projects, do you have an exception or something ?